platform/ios/src/MGLMapView.mm

```diff
@@ -389,8 +389,14 @@ - (nonnull NSURL *)styleURL
     }
 
     NSString *styleURLString = @(self.mbglMap.getStyle().getURL().c_str()).mgl_stringOrNilIfEmpty;
-    MGLAssert(styleURLString, @"Invalid style URL string %@", styleURLString);
-    return styleURLString ? [NSURL URLWithString:styleURLString] : nil;
+
+    if (!styleURLString)
+    {
+        MGLLogInfo(@"Warning: Invalid style URL string %@", styleURLString);
```

Comment:
Seeing `MGLLogInfo` followed by “Warning” feels a little incongruous — are you comfortable with how/when this message appears when using `MGLLogInfo` (as opposed to `MGLLogError`, perhaps)?

Reply:
Funny you say this, as I was thinking the same thing. Given that we don't have a `warning` level (I think we should), this should probably be logged as an error.

Reply:
@julianrex could we add `MGLLogWarning` before landing this pr? how hard would it be?